lnrpc/rpc.proto

```diff
@@ -2909,12 +2909,39 @@ message ListPaymentsRequest {
     that pending payments, as well as failed payments will show up if this
     field is set to True.
     */
-    bool include_incomplete = 1;
+    bool include_incomplete = 1 [json_name = "include_incomplete"];
+
+    /**
+    The index of a payment that will be used as either the start or end of a
+    query to determine which payments should be returned in the response.
+    */
+    uint64 index_offset = 2 [json_name = "index_offset"];
+
+    /// The max number of payments to return in the response to this query.
+    uint64 num_max_payments = 3 [json_name = "num_max_payments"];
```

Comment:
`num_max_payments`, what is that the abbreviation of? I read this as 'number of maximum payments'.

Reply:
Changed it to `max_payments` and `NumMaxPayments`->`MaxPayments`. In listinvoices we have `max_invoices`.